images/manageiq-ui-worker/Dockerfile

```diff
@@ -6,16 +6,22 @@ MAINTAINER ManageIQ https://manageiq.org
 
 ARG NPM_REGISTRY_OVERRIDE
 
+ARG ARCH=amd64
```

Comment:
Same here, I would prefer the default to be `x86_64`

Reply:
okay